scripts/makei

```diff
@@ -207,7 +207,17 @@ def handle_compile(args):
         filenames = [args.file]
     elif args.files:
         filenames = map(os.path.basename, args.files.split(':'))
-    targets = get_compile_targets_from_filenames(filenames)
+    targets = []
+    source_names = []
+    for name in filenames:
+        if os.path.isdir(name):
+            targets.append(make_dir_target(name))
+        else:
+            source_names.append(name)
+    #print("source:"+' '.join(source_names))
+    #print("compile targets:"+' '.join(get_compile_targets_from_filenames(source_names)))
```

Comment:
Can we remove these comments?

Reply:
These were here for debugging purposes, so I would prefer to leave them.